src/modules/saveData.js

```diff
@@ -0,0 +1,18 @@
+// local storage
+export default function saveData() {
+  // Create an array of task objects from the list items
+  const tasksArray = [];
+  const listItems = document.querySelectorAll('#list-container li');
+
+  listItems.forEach((item, index) => {
+    const task = {
+      description: item.textContent,
+      index: index + 1,
+      completed: item.classList.contains('checked'),
+    };
+    tasksArray.push(task);
+  });
+
+  // Convert the tasks array to a JSON string and store it in local storage
+  localStorage.setItem('tasks', JSON.stringify(tasksArray));
```

Comment:
Currently, your changes are being saved to local storage but are not being reapplied to your app when the page is refreshed. Kindly make sure that all changes made to the app are persisted across page refreshes🙏

Reply:
done